pkgs/by-name/gr/grip-grab/package.nix

```
@@ -0,0 +1,30 @@
+{
+  lib,
+  rustPlatform,
+  fetchFromGitHub,
+}:
+let
+  pname = "grip-grab";
+  version = "0.3.0";
+in
+rustPlatform.buildRustPackage {
+  inherit pname version;
+
+  src = fetchFromGitHub {
+    owner = "alexpasmantier";
+    repo = "grip-grab";
+    rev = "refs/tags/v${version}";
+    hash = "sha256-5nKDeJ5d9ECd9zvSK22zN15m8qCt/L3VKem9hVQzLzU=";
```

```suggestion
  version = "0.6.7";
in
rustPlatform.buildRustPackage {
  inherit pname version;

  src = fetchFromGitHub {
    owner = "alexpasmantier";
    repo = "grip-grab";
    tag = "v${version}";
    hash = "sha256-e7duLL4tjW+11jXUqU6sqoKTAPGkH81iDCfjtNcnd4I=";
```
This also resolves the auto-upgrade functionality (it is disabled by default).